pkg/gitprovider/bitbucket.go

```diff
@@ -108,14 +111,14 @@ func (g *BitbucketGitProvider) GetRepoBranches(repositoryId string, namespaceId
 	client := g.getApiClient()
 	var response []*GitBranch
 
-	owner, repo, err := g.getOwnerAndRepoFromFullName(repositoryId)
-	if err != nil {
-		return nil, err
-	}
+	// owner, repo, err := g.getOwnerAndRepoFromFullName(namespaceId)
+	// if err != nil {
+	// 	return nil, err
+	// }
```

Comment:
Ok, I thought the bug was from our side, but I can see now that you've changed `repositoryId` as an argument and put `namespaceId` instead. 
This is what's causing the issue for you so please, uncomment this section and put `repositoryId` as an argument instead.
Also, what was the reason for changing this and then commenting it?

Reply:
Also, refer to the comment above and implement the `prebuild scope` for bitbucket. The example above is how it is for github.

Reply:
> Ok, I thought the bug was from our side, but I can see now that you've changed `repositoryId` as an argument and put `namespaceId` instead. This is what's causing the issue for you so please, uncomment this section and put `repositoryId` as an argument instead. Also, what was the reason for changing this and then commenting it?

I followed your instructions, but I'm still encountering the same error! To my knowledge, I think I was attempting to log what the `getOwnerAndRepoFromFullName` function does, which might have caused the `repositoryId` to be changed to `namespaceId`.

![Screenshot from 2024-09-05 23-54-23](https://github.com/user-attachments/assets/8e9fe9e6-1727-42e1-93a0-62cf4ac59250)

Reply:
> Also, refer to the comment above and implement the `prebuild scope` for bitbucket. The example above is how it is for github.

It's done! I will push the changes once you confirm about commenting that part of code.

Reply:
Please push the changed code with uncommenting that part and reverting back to repositoryId so I can test it as well. 
Alongside that, provide me with information what OS are you using? Thanks.

Reply:
Ok, on way way! Welcome!
My OS info:
```
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.4 LTS
Release:        22.04
Codename:       jammy
```

Reply:
I've also included the prebuild scope for GitLab because I had previously implemented GitLab webhook support and hadn't added the scope at that time.